src/install/Task.zig

```diff
@@ -0,0 +1,380 @@
+//! Schedule long-running callbacks for a task
```

Comment:
`PackageManagerTask.zig` instead of `Task.zig`? Task.zig is currently the EventLoop's tasks